samplers/aws/xray/go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/go-logr/logr v1.2.3
 	github.com/go-logr/stdr v1.2.2
 	github.com/stretchr/testify v1.8.1
+	go.opentelemetry.io/contrib/propagators/aws v1.12.0
```

Comment:
Please ensure `make crosslink` is run to add replace statements for modules in this repo.

Reply:
done